### PR TITLE
[osx] bump minimum version to 10.9

### PR DESCRIPTION
**Pull request by Rechi**
## Motivation and Context
Get rid of the `IsMavericksOrHigher` by setting Mavericks (10.9) as minimum version.

## How Has This Been Tested?
complete recompile of kodi including depends for osx

## Types of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by Memphiz — CHANGES_REQUESTED** [review]
Fine by me. Though I would like to see the removal of „IsMavericksOrHigher“ in a separate commit instead of having it buried in the deployment target bump.

Thx for the cleanups :)

**Comment by Rechi** [comment]
@Memphiz updated

**Review by Memphiz — APPROVED** [review]

**Comment by Memphiz** [comment]
Thx - fine by me.

File: xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp [hunk 1]
```diff
@@ -278,13 +277,10 @@ bool CLinuxRendererGL::Configure(const VideoPicture &picture, float fps, unsigne
...
   // effect when rendering with pbo
   // when used on intel gpu - we have to quirk it here
-  if (CDarwinUtils::IsMavericksOrHigher())
-  {
-    std::string rendervendor = CServiceBroker::GetRenderSystem()->GetRenderVendor();
-    StringUtils::ToLower(rendervendor);
-    if (rendervendor.find("intel") != std::string::npos)
-      m_pboSupported = false;
-  }
+  std::string rendervendor = CServiceBroker::GetRenderSystem()->GetRenderVendor();
```
Comment (FernetMenta) [inline xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp]: please drop this hack. If PBOs do not work, we have to fix if.

**Review by FernetMenta — CHANGES_REQUESTED** [review]
please drop the PBO hack. Other than that looks good. Thanks!

**Comment by Rechi** [comment]
@FernetMenta updated

**Review by FernetMenta — APPROVED** [review]

**Pull request merged**